maas/testing/generate-docs.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+# Copyright 2015, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import collections
+import json
+import tabulate
+import re
+
+
+def print_tables(data, data_type=None):
+    for host, lines in data.viewitems():
+        if lines:
+            title = '{dt} - {host}'.format(dt=data_type, host=host)
+            print title
+            print '-' * len(title)
+            print tabulate.tabulate(lines, headers='keys', tablefmt='rst')
+            print
+
+if __name__ == '__main__':
```

Comment:
some of this should probably be in it's own function as best practice, also, docstrings or comments?